zeppelin-plugins/launcher/yarn/src/main/java/org/apache/zeppelin/interpreter/launcher/YarnRemoteInterpreterProcess.java

```diff
@@ -178,6 +178,10 @@ public void start(String userName) throws IOException {
       this.appId = appResponse.getApplicationId();
       ApplicationSubmissionContext appContext = yarnApplication.getApplicationSubmissionContext();
       appContext = createApplicationSubmissionContext(appContext);
+      String YARN_NODE_LABEL_EXPRESSION = "yarn.node.label.expression";
```

Comment:
It is better to name it as `zeppelin.interpreter.yarn.node.label.expression` to make it consistent with memory and core configuration (`zeppelin.interpreter.yarn.resource.memory` & `zeppelin.interpreter.yarn.resource.cores`).

Besides that, the document also needs to be updated.

Reply:
Thanks @zjffdu for the reviews.   It's done.